CHANGELOG.md

```diff
@@ -6,10 +6,14 @@
   * Limit ping request body size to 1MB. ([#1098](https://github.com/mozilla/glean/pull/1098))
 * iOS
   * Implement ping tagging (i.e. the `X-Source-Tags` header) through custom URL ([#1100](https://github.com/mozilla/glean/pull/1100)).
+  * Update `glean_parser` to `v1.28.3`
+    * BUGFIX: Generate valid C# code when using Labeled metric types.
+    * BUGFIX: Support `HashSet` and `Dictionary` in the C# generated code.
```

Comment:
Probably a merge issue -- but this is appearing under the iOS section, but relates to C#

Reply:
Good catch!